code/modules/mob/living/simple_animal/abnormality/waw/generalb.dm

```diff
@@ -38,6 +38,8 @@
 	var/fire_cooldown
 	var/fireball_range = 30
 	var/volley_count
+	//If the general has her post breach icon.
+	var/static/true_breached = FALSE
```

Comment:
Why the static var?

Reply:
If she breaches and she dies all of her variables will reset because she is initialized again. Static makes it so that this variable stays and is shared across all general bees after this point.